### PR TITLE
pi-appliance doc update

### PR DESCRIPTION
**Pull request by laclaro**
related to #1967.

Reviewing the pi-appliance tool I felt like updating the documentation in this regard.

### CODE REVIEW

File: doc/configuration/privacyidea-appliance.rst [hunk 1]
```diff
@@ -112,14 +112,56 @@ Restore
...
+
+.. figure:: images/appliance/redundancy-successful.png
+   :width: 450px
+
+.. _pi-appliance_updates:
+
+Updates
+.......
+
+In this menu, you can setup cronjobs for automatic updates which is seldomly
```
[the remark this answers is not in the document]
Comment (laclaro) [reply doc/configuration/privacyidea-appliance.rst:153]: i think this is correct

[the remark this answers is not in the document]
Comment (plettich) [reply doc/configuration/privacyidea-appliance.rst:153]: According to [merriam-webster](https://www.merriam-webster.com/dictionary/seldomly) this is *archaic*. You can use *seldom* as an adjective as well as an adverb.

File: doc/configuration/privacyidea-appliance.rst [hunk 2]
```diff
@@ -112,14 +112,56 @@ Restore
...
 You can select a backup and you are asked if you want to restore the data. 
 
 .. warning:: Existing data is overwritten and will be lost.
 
+.. _database:
+
+Database: Setup Redundancy
+..........................
+
+The appliance-tool is also capable of setting up a redundant setup, between
```
Comment (plettich) [inline doc/configuration/privacyidea-appliance.rst:128]: no "," between "setup" and "between".

File: doc/configuration/privacyidea-appliance.rst [hunk 3]
```diff
@@ -112,14 +112,56 @@ Restore
...
+two privacyIDEA nodes. For the setup, the privacyIDEA slave instance has to
+be setup and running with identical software than the master instance. The
+appliance-tool uses a tinc-tunnel to share the configuration and the database.
```
Comment (plettich) [inline doc/configuration/privacyidea-appliance.rst:131]: "can" use a tinc tunnel?

File: doc/configuration/privacyidea-appliance.rst [hunk 4]
```diff
@@ -112,14 +112,56 @@ Restore
...
+
+.. figure:: images/appliance/database.png
+   :width: 400px
+
+.. note:: The redundancy requires to setup a tinc VPN connection between
```
Comment (plettich) [inline doc/configuration/privacyidea-appliance.rst:136]: I don't think the tinc tunnel is required, will check.

File: doc/configuration/privacyidea-appliance.rst [hunk 5]
```diff
@@ -112,14 +112,56 @@ Restore
...
+.. figure:: images/appliance/redundancy-successful.png
+   :width: 450px
+
+.. _pi-appliance_updates:
+
+Updates
+.......
+
+In this menu, you can setup cronjobs for automatic updates which is seldomly
+used in productive setups.
```
Comment (plettich) [inline doc/configuration/privacyidea-appliance.rst:154]: actually i wouldn't put that into the docs...

File: doc/configuration/privacyidea-appliance.rst [hunk 6]
```diff
@@ -112,14 +112,56 @@ Restore
...
+be setup and running with identical software than the master instance. The
+appliance-tool uses a tinc-tunnel to share the configuration and the database.
+
+.. figure:: images/appliance/database.png
+   :width: 400px
+
+.. note:: The redundancy requires to setup a tinc VPN connection between
+   the nodes and an SSH root login. Make sure the services are installed.
+
+.. note:: Keep in mind that the log files and backups are also synchronized between
```
Comment (plettich) [inline doc/configuration/privacyidea-appliance.rst:139]: The stuff in `/var/log/privacyidea/` and `/var/lib/privacyidea/backup`? Didn't know, will check. It doesn't need to to that.

Comment (plettich) [reply doc/configuration/privacyidea-appliance.rst:139]: And it does not. The logs and backups are local to each node.

File: doc/configuration/privacyidea-appliance.rst (hunk 4 above)
Comment (plettich) [reply doc/configuration/privacyidea-appliance.rst:136]: tinc tunnel is not required.

File: doc/configuration/privacyidea-appliance.rst [hunk 7]
```diff
@@ -112,14 +112,56 @@ Restore
...
+   the nodes. The setup of log rotation and backup is usually only neccessary on
```
[the remark this answers is not in the document]
Comment (laclaro) [reply doc/configuration/privacyidea-appliance.rst]: done

File: doc/configuration/privacyidea-appliance.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (laclaro) [reply doc/configuration/privacyidea-appliance.rst:153]: okay

File: doc/configuration/privacyidea-appliance.rst (hunk 2 above)
Comment (laclaro) [reply doc/configuration/privacyidea-appliance.rst:128]: done

File: doc/configuration/privacyidea-appliance.rst (hunk 4 above)
Comment (laclaro) [reply doc/configuration/privacyidea-appliance.rst:136]: okay

File: doc/configuration/privacyidea-appliance.rst (hunk 3 above)
Comment (laclaro) [reply doc/configuration/privacyidea-appliance.rst:131]: done

File: doc/configuration/privacyidea-appliance.rst (hunk 5 above)
Comment (laclaro) [reply doc/configuration/privacyidea-appliance.rst:154]: I put it to have all the menu top items covered. We will move the pi-appliance documentation to the appliance tool repository.

File: doc/configuration/privacyidea-appliance.rst (hunk 6 above)
Comment (laclaro) [reply doc/configuration/privacyidea-appliance.rst:139]: okay, deleted

Comment (laclaro) [reply doc/configuration/privacyidea-appliance.rst:139]: I deleted the note and added another to the audit rotation section.

**Comment by laclaro** [comment]
Requested Conelius instead of Paul for a quick check.

File: doc/configuration/privacyidea-appliance.rst [hunk 8]
```diff
@@ -112,14 +112,54 @@ Restore
...
 
 .. warning:: Existing data is overwritten and will be lost.
 
+.. _database:
+
+Database: Setup Redundancy
+..........................
+
+The appliance-tool is also capable of setting up a redundant setup between
+two privacyIDEA nodes. For the setup, the privacyIDEA slave instance has to
```
Comment (laclaro) [inline doc/configuration/privacyidea-appliance.rst:129]: slave = second

File: doc/configuration/privacyidea-appliance.rst [hunk 9]
```diff
@@ -112,14 +112,54 @@ Restore
...
+be setup and running with identical software than the master instance. The
```
Comment (laclaro) [inline doc/configuration/privacyidea-appliance.rst:130]: setup? only packages have to be installed

File: doc/configuration/privacyidea-appliance.rst [hunk 10]
```diff
@@ -112,14 +112,54 @@ Restore
...
+
+Audit Rotation
+..............
+
+In the `Audit Rotation` menu, you can setup cronjobs for the audit rotation conditioned
+by age or the number of entries. The syntax follows the crontab syntax as explained
+in :ref:`backup_and_restore`.
+
+.. note:: Keep in mind that the audit log is synchronized between the nodes in a redundant
+   setup. Therefore an audit log rotation is only necessary on one node.
```
Comment (cornelinux) [inline doc/configuration/privacyidea-appliance.rst:162]: Therefore you should not run an audit rotation on two nodes at the same time!

File: doc/configuration/privacyidea-appliance.rst (hunk 9 above)
Comment (cornelinux) [inline doc/configuration/privacyidea-appliance.rst:130]: For the setup, you only need to install the privacyIDEA packages in the same version on the second node. It is not necessary to configure anything on the second node.

File: doc/configuration/privacyidea-appliance.rst (hunk 10 above)
Comment (laclaro) [reply doc/configuration/privacyidea-appliance.rst:162]: ok

File: doc/configuration/privacyidea-appliance.rst (hunk 9 above)
Comment (laclaro) [reply doc/configuration/privacyidea-appliance.rst:130]: I rewrote the paragraph.

**Pull request merged**